ios-target/Extensions/UIButtonExtension.swift

```diff
@@ -11,8 +11,8 @@ import UIKit
 
 extension UIButton {
   
-  func enableButton(enable: Bool) {
-    self.alpha = enable ? 1 : 0.5
-    self.isEnabled = enable
+  func enableButton(_ enable: Bool) {
```

Comment:
since it's an extension of `UIButton` already, why don't you name it just `enable(_ enabled: Bool)`?